timefold-solver-python-core/src/main/python/score/_constraint_stream.py

```diff
@@ -2083,6 +2148,39 @@ def concat(self, other):
         else:
             raise RuntimeError(f'Unhandled constraint stream type {type(other)}.')
 
+    def complement(self, item_type: Type[A], padding_function_b: Callable[[A], B] = None,
+                   padding_function_c: Callable[[A], C] = None,
+                   padding_function_d: Callable[[A], D] = None) -> 'QuadConstraintStream[A, B, C, D]':
```

Comment:
Hmm, and now I realize that this is wrong too.
I cannot use `None` here, because that'd mean the java `complement` gets `null`, instead of the correct overload being called.
So it seems that I have to re-implement every Java overload as a Python delegate. Sigh.

Reply:
Remember: Python **does not** have function overloading. So you must have a single function that delegate to all possible Java overloads.